Documentation/benchmarks/etcd-storage-memory-benchmark.md

```diff
@@ -0,0 +1,97 @@
+# Storage Memory Usage Benchmark
+
+The storage(todo: link to design doc about storage) has two components that consume physical memory: the _in-memory index_, and the _page cache_ for on-disk data.
```

Comment:
1. "etcd storage has two components..."
2. Can we either resolve the TODO before merging/publishing, or move it out of the main text and into a line comment or something?